static/src/javascripts-legacy/projects/commercial/modules/article-aside-adverts.js

```diff
@@ -15,58 +13,57 @@ define([
     config,
     mediator,
     fastdom,
-    addSlot,
-    createSlot,
     commercialFeatures
 ) {
     var minArticleHeight = 1300;
     var minImmersiveArticleHeight = 600;
 
+    var mainColumnSelector = '.js-content-main-column';
```

Comment:
Still in master at the moment?? https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/commercial/modules/article-aside-adverts.js#L37